backend/src/models/index.js

```diff
@@ -1,7 +1,27 @@
 const Region = require('./Region');
+const Hierarchy = require('./Hierarchy');
+const HierarchyNames = require('./HierarchyNames');
+
+Hierarchy.belongsToMany(Region, {
+    through: 'hierarchy_region_mapping', // Name of the intermediate mapping table
+    foreignKey: 'alt_region_id', // Foreign key in the mapping table referring to AlternativeHierarchy
+    otherKey: 'region_id', // Foreign key in the mapping table referring to Region
+});
+
+Hierarchy.belongsTo(HierarchyNames, {
+    foreignKey: 'hierarchy_id',
+});
+
+Region.belongsToMany(Hierarchy, {
+    through: 'hierarchy_mapping', // Name of the intermediate mapping table
+    foreignKey: 'region_id', // Foreign key in the mapping table referring to Region
+    otherKey: 'alt_region_id', // Foreign key in the mapping table referring to Hierarchy
```

Comment:
Looks like a good catch... I'm curious now: how did it work when I tested it?
Why did this work?
```js
    const hierarchyEntry = await Hierarchy.findByPk(regionId, {
        include: [{
            model: Region,
            through: {
                attributes: []
            }, // Exclude the attributes of the join table
        }]
    });
```

Reply:
I guess it worked because I also have this:
```js
Hierarchy.belongsToMany(Region, {
    through: 'hierarchy_region_mapping', // Name of the intermediate mapping table
    foreignKey: 'alt_region_id', // Foreign key in the mapping table referring to AlternativeHierarchy
    otherKey: 'region_id', // Foreign key in the mapping table referring to Region
});
```
But do I really need both of these belongsToMany calls?